Rakefile

```diff
@@ -9,5 +9,5 @@ Yast::Tasks.configuration do |conf|
   conf.obs_sr_project = nil
   # TODO: improve it, at least do not get worse
   # TODO: remove condition when new packaging tasks are accepted to factory
-  conf.documentation_minimal = 50 if conf.respond_to?(:documentation_minimal=)
+  conf.documentation_minimal = 90 if conf.respond_to?(:documentation_minimal=)
```

Comment:
@mvidner well, it was actually discussed several yeards on yardoc and they suggest see https://github.com/lsegal/yard/issues/720 but I agree that it do not see stuff in difference object. 
In general I prefer to stick with common `# (see AbstractWidget#label)` as solve general problem with different project, as it sucks also with generating links what methods are inherited and documented.

Compare http://www.rubydoc.info/github/yast/yast-bootloader/Bootloader/ConsoleWidget ( missing inherits methods from CustomWidget) and http://www.rubydoc.info/github/yast/yast-yast2/CWM/Tree ( all inherits methods ).

So we should use see and hope that in reasonable future it will be possible to somehow generate whole documentation linked together.